pkg/topology/kademlia/kademlia.go

```diff
@@ -437,6 +441,25 @@ func (k *Kad) manage() {
 		}
 	}()
 
+	k.wg.Add(1)
+	go func() {
+		defer k.wg.Done()
+		for {
+			select {
+			case <-k.halt:
+				return
+			case <-k.quit:
+				return
+			case <-time.After(1 * time.Second):
```

Comment:
this might be a bit aggressive, maybe wait longer here?

Reply:
since the metric is averaged, it takes a while for it to actually add up and show up in the measurement, and if the peer has a high latency it will anyway take a long time until it is actually reflected in the smoothed metric (it has a delay), so if we poll more slowly the delay will become much more significant (it would be a few seconds as it is, which is already quite slow). also, the actual pinging is not a costly operation (it is just a couple of kilobytes for 200 peers) also on the downstream peer, so I think it is ok as is